### ISSUE TITLE
Client-side support (Support for IndexDB / Localstorage / Remotestorage)

### ISSUE THREAD

**Issue by marcelklehr** [geddy/model#153]
It would be nice to switch an app's backend from server-side to client-side more or less seamlessly.

This allows one to switch to unhosted stuff like grimwire, and remotestorage!

**Comment by jrhicks** [comment]
@der-On Is anyone still working to bring Model to the front-end?

**Comment by der-On** [comment]
@jrhicks yes. I'm currently using the rest-adapter in my second production project. I need to write an example on how to integrate it correctly. It's not as straight forward as on the server.

**Comment by der-On** [comment]
@jrhicks I've now added docs about how to use the rest-adapter in the browser: https://github.com/geddy/model#using-rest-adapter-in-browser

**Comment by jrhicks** [comment]
thanks @der-On
